elastic/kibana
# [Embeddables Rebuild] Remove legacy examples

PR does the following
1. removes legacy example embeddables
2. Replaces FilterDebugger legacy embeddable with react embeddable
3. Moves FilterDebugger react embeddable to portable_dashboards_example since that is where its used
4. Fixes DashboardContainer.addNewPanel to return ApiType for react embeddables.

### test instructions
1. start kibana `yarn start --run-examples`
2. Open kibana menu and click "Developer examples"
3. Use developer examples search bar to find "Portable Dashboards" example and click on the card
4. Verify first example that shows control filters works
